ddtrace/appsec/utils.py

```diff
@@ -7,7 +7,8 @@
 def _appsec_rc_features_is_enabled():
     # type: () -> bool
     if asbool(os.environ.get("DD_REMOTE_CONFIGURATION_ENABLED", "false")):
-        return APPSEC_ENV not in os.environ
+        return True
+        #return APPSEC_ENV not in os.environ
```

Comment:
@juanjux I've had to edit this as I could not make it work with both 1-click activation and custom rules; worth looking into it more deeply when this becomes a real feature

Reply:
So this didn't work even with DD_REMOTE_CONFIGURATION_ENABLED set to true and DD_APPSEC_ENABLED too?